manifests/virtualenv.pp

```diff
@@ -97,11 +97,18 @@
     if (( versioncmp($_virtualenv_version,'1.7') > 0 ) and ( $systempkgs == true )) {
       $system_pkgs_flag = '--system-site-packages'
     } elsif (( versioncmp($_virtualenv_version,'1.7') < 0 ) and ( $systempkgs == false )) {
-      $system_pkgs_flag = '--no-site-packages'
+      if $facts['os']['release']['full'] == '20.04' {
```

Comment:
I might suggest that this conditional logic here put into moduledata instead.  This is only a suggestion to clean this up though.

Reply:
There is no module hieradata yet.

Reply:
Would be easy to add if you have the time.

Reply:
Sure this is a good idea, but I am currently too busy with adding other 20.04 support. And it is also not part of the PR to change the params.pp to module hieradata. Ths should be done in another PR/Ticket.